Automattic/jetpack
# Remove old Fly Panel for showing in some scenarios

Fixes https://github.com/Automattic/dotcom-forge/issues/6809

## Proposed changes:
Old fly panel was showing when clicking on WP & Profile in the masterbar when Jetpack is enabled but SSO is disabled.

> [!IMPORTANT]
> This PR should be merged after this diff (D151708-code) is merged since it remove the function of the Fly panel and this PR removes content.

Before:

https://github.com/Automattic/jetpack/assets/402286/2dfef4ca-c5c4-48b0-86c8-965669b33007

After:

https://github.com/Automattic/jetpack/assets/402286/60421d0e-1f73-4780-9682-74495f4b5f75

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Apply this diff: D151708-code
* Sandbox `s0.wp.com`
* Disable SSO in Jetpack Settings
* Click on W logo
* It should link to `My Home` at WPcom
* Click on the profile picture
* It should show a fly panel with the Profile & Logout link
* Check if anything is broken on self-hosted sites
* Check if everything works in mobile (Menu open, logout, profile link)
* Check if anything is broken with SSO enabled
* Check if anything is broken with Classic interface is enabled (wp-admin)
